### PR TITLE
added: button & alert components

### PR DESCRIPTION
**Pull request by sharmayatendra**

### CODE REVIEW

File: components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+.link-secondary {
+    color: var(--secondary-link-color);
+}
+
+.flex {
+    display: flex;
+    align-items:center;
+}
+
+.icon-cart,.icon-wishlist {
```
Comment (xii-vi) [inline components/Button/button.css]: Instead of using two different class try to use single one if possible.

File: components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    background-color: var(--icon-btn-color1);
+    color: var(--btn-color);
+}
+
+.icon-btn-wishlist {
+    background-color: var(--icon-btn-color2);
+    color: var(--btn-color);
+}
+
+.icon-btn-cart:hover,.icon-btn-wishlist {
```
Comment (xii-vi) [inline components/Button/button.css]: As mentioned in line 60

**Review by xii-vi — COMMENTED** [review]
Added two comments. Everything else is fine.

File: components/Alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                <li class="nav-links"><a href="/components/Avatar/avatar.html">Avatar</a></li>
+                <li class="nav-links"><a href="#">Alert</a></li>
+                <li class="nav-links"><a href="/components/Badge/badge.html">Badge</a></li>
+                <li class="nav-links"><a href="#">Button</a></li>
+                <li class="nav-links"><a href="#">Card</a></li>
+                <li class="nav-links"><a href="#">Image</a></li>
+                <li class="nav-links"><a href="#">Input</a></li>
+                <li class="nav-links"><a href="#">List</a></li>
+                <li class="nav-links"><a href="#">Navigation</a></li>
+                <li class="nav-links"><a href="#">Toast</a></li>
```
Comment (viru2001) [inline components/Alert/alert.html]: you can add navigation links of each component here ( or for at least the components you made ).

File: components/Button/button.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
```
Comment (viru2001) [inline components/Button/button.html:8]: you can import this in your main CSS file so that you don't need to import it again and again .

**Review by viru2001 — COMMENTED** [review]
Everything looks good to me . I suggested small changes .

File: components/Button/button.css (hunk 2 above)
Comment (sharmayatendra) [reply components/Button/button.css]: thanks for info..done with the changes

File: components/Alert/alert.html (hunk 3 above)
Comment (sharmayatendra) [reply components/Alert/alert.html]: done the changes

**Review by xii-vi — APPROVED** [review]
Now you can merge.

**Pull request merged**